src/widget/form/chatform.h

```diff
@@ -54,6 +54,7 @@ class ChatForm : public GenericChatForm
 signals:
     void sendFile(uint32_t friendId, QString, QString, long long);
     void aliasChanged(const QString& alias);
+    void invitedCall(int friendId);
```

Comment:
The `frendId` variable must be typed `uint32_t`. Btw. I prefer the typedef `quint32` within c++ api, but that's only a question of taste. :)